electron_app/src/webcontents-handler.js

```diff
@@ -84,14 +84,6 @@ function _CutCopyPasteSelectContextMenus(params) {
     }];
 }
 
-function onSelectedContextMenu(ev, params) {
```

Comment:
THis moved to a place where I can use IPC as I need it 1. without spellcheck stuff and 2. with spellcheck stuff. As I cannot add the spellcheck options in Main I have to access it renderer. At this point I actually could fully move it to the renderer to keep stuff smaller.